acceptance/pe/tests/basic.rb

```diff
@@ -0,0 +1,21 @@
+require 'scooter'
+
+test_name "basic validation of puppetdb-cli subcommands" do
```

Comment:
@ajroetker do you want to set this test to skip until the ability to use tokens is resolved? Just add the following after this line.
```
skip_test "This test is blocked by PDB-xxxx
```

We know that the test is doing its job. With the skip, I think we can merge this PR and massage the rake task namespace for PE afterward to make it match the other tools.

cc: @zreichert